migrations/versions/2023_06_11_09_09_00_8337558f4f36_move_deleted_utc_into_state_user_.py

```diff
@@ -0,0 +1,46 @@
+"""Move 'deleted_utc' into 'state_user_deleted_utc'.
+
+Revision ID: 8337558f4f36
+Revises: 6403c9151c12
+Create Date: 2023-06-11 09:09:00.235602+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '8337558f4f36'
+down_revision = '6403c9151c12'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('comments', sa.Column('state_user_deleted_utc', sa.DateTime(), nullable=True))
+    op.drop_column('comments', 'deleted_utc')
```

Comment:
This will un-delete all currently deleted comments.

The SQL to do this and the previous comment correctly is something like:
```sql
ALTER TABLE comments ALTER COLUMN deleted_utc TYPE timestamp with time zone USING to_timestamp(deleted_utc);
ALTER TABLE comments RENAME COLUMN deleted_utc TO state_user_deleted_utc;
```

Reply:
Dammit, could've sworn I'd written update code. I think it got accidentally nuked when I was tinkering with the changelist >_<

Anyway, fixed.